.github/workflows/ci.yml

```diff
@@ -2,20 +2,21 @@ name: CI
 on:
     push:
         branches:
-            - main
+            - '*'
```

Comment:
We don't want to run CI on non-PR branches since we have limited concurrent resources.

Reply:
Reverted.